configure.ac

```diff
@@ -81,6 +81,15 @@ AC_ARG_ENABLE([direct],
 	[],
 	[enable_direct=no])
 
+AC_ARG_WITH([verbs-xrc],
+    AC_HELP_STRING([--with-verbs-xrc],
+		   [Enable Verbs XRC transport support @<:@default=no@:>@]))
+
+if test "$with_verbs_xrc" != "" && test "$with_verbs_xrc" != "no"; then
+	AC_DEFINE([INCLUDE_VERBS_XRC], 1,
+		  [Define to 1 to enable Verbs XRC transport support])
+fi
```

Comment:
I'd like to build in xrc support always

Reply:
Ok. The reason it was done this way was to ease XRC into acceptance. That is, if one had no interest in possibly using XRC then they could not configure it in and the RC MSG_EP and data path code would not be impacted. To make this change, we will then need to re-organize the code and it will eliminate the ifdefs, but it also impacts the loads in the RC data path, so that will need to be addressed.

Reply:
This done now. Modified the fi_ibv_ep so the endpoint size is not impacted by including XRC by default in the build.